pyfr/integrators/writers.py

```diff
@@ -1,17 +1,16 @@
 # -*- coding: utf-8 -*-
 
-from abc import abstractmethod
 import itertools as it
 import os
-
+import h5py
+from mpi4py import MPI
```

Comment:
As importing mpi4py can — if you are not very careful — result in MPI_Init being called.  This gets us into all kinds of trouble as when running a simulation we need to ensure that we prefork first.  When post-processing we do not want MPI at all.  Hence, we always import MPI locally to avoid the possibility of this.